docs/src/content/ignition/docs/advanced/_dirinfo.yaml

```
@@ -3,5 +3,6 @@ section-title: Advanced
 order:
   - /execution
   - /deployment-artifacts
+  - /migrating
   - /reconciliation
   - /versioning
```

```suggestion
order:
  - /execution
  - /deployment-artifacts
  - /reconciliation
  - /versioning
  - /migrating
```

I think it is its own thing so lets put it at the end.